packages/utils/src/money/format-balance/format-balance.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { formatBalance } from './format-balance';
```

Comment:
Migrating `formatBalance` from the extension and getting claude to generate these tests. We do some specific formatting here, it could be good to refactor these tests with more detail on the business logic. 

I'll need to refactor + remove from the extension